crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -171,8 +174,20 @@ pub async fn daemon_server(
         }
         CloseReason::Interrupt
     });
-    // TODO: be more methodical about this choice:
-    let cookie_dir = base.repo_root.join_component(".git");
+    // We already store logs in .turbo and recommend it be gitignore'd.
```

Comment:
I'm fine with this, but we should note that we only add `.turbo` to people's gitignore when they run `turbo link`, so if someone adds turbo and it starts the daemon, then they might end up accidentally committing this cookie dir.

Reply:
We also write cached logs there: https://github.com/vercel/turbo/blob/2bb8c9a2477b5fb20b4711fe34412559bc85aab9/cli/internal/graph/graph.go#L241